node/rest/src/lib.rs

```diff
@@ -181,6 +182,8 @@ impl<N: Network, C: ConsensusStorage<N>, R: Routing<N>> Rest<N, C, R> {
 
             // GET ../program/..
             .route(&format!("/{network}/program/:id"), get(Self::get_program))
```

Comment:
Is there a way to return the edition from the original `get_program` endpoint? If not, we may want to consider adding an optional `Metadata` to the query that surfaces it, and potentially other info like owner, tx_id, etc.

Reply:
I'm not sure you can without breaking the API.
The `Metadata` option seems reasonable.

Reply:
I added the option for metadata, returning the edition and TX ID.
I did not add `ProgramOwner`, since it is not used for anything.